.rubocop.yml

```diff
@@ -0,0 +1,126 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
Now the minimum ruby version required to run Sinatra is `2.3` so this could be updated to reflect on that WDYT?

https://github.com/sinatra/sinatra/blob/master/sinatra.gemspec#L49

Reply:
This will probably not happen before Sinatra 3.0, so we can set this to `2.6`